hickle/tests/test_bitshuffle.py

```diff
@@ -0,0 +1,65 @@
+import os
+import sys
+import numpy as np
+import pytest
+import hickle as hkl
+from hickle.hickle_bshuf import BitShuffleLz4
+
+
+H5_FILE = "./test.h5"
+array1 = np.random.rand(16, 1, 1048576)
+
+
+def install_hdf5plugin():
+    cmd = "python3 -m pip install -U hdf5plugin"
+    exit_code = os.system(cmd)
+    assert exit_code == 0
+
+
+def uninstall_hdf5plugin():
+    cmd = "python3 -m pip uninstall -y hdf5plugin"
+    exit_code = os.system(cmd)
+    assert exit_code == 0
+
+
+def validator(label, array_A, array_B):
+    print("validator: {} .....".format(label))
+    
+    # Check the two are the same file
+    print(array_A.shape, array_B.shape)
+    assert array_A.shape == array_B.shape
+    print(array_A.dtype, array_B.dtype)
+    assert array_A.dtype == array_B.dtype
+    assert np.all((array_A, array_B))
+
+
+@pytest.mark.no_compression
+@pytest.mark.skipif(sys.platform == "win32" and sys.maxsize < 2**32, reason="no wheel for hdf5plugin available on windows 32 bit")
+@pytest.mark.order(1)
```

Comment:
please remove any `pytest_order markrs and stuff again. Your test does not make sense to run before not all components of hickle have been thoroughly unit tested. In other words not before all tests in `test_99_hickle.py` and any tests before have been successful as not until then it is assured that `hickle.dump` and `hickle.load` work as expected both of which your Interface relies upon. An any of both triggering an error either directly or cause any of the sub components used by them is broken will cause your tests to fail too. `pytest_order` is not an option as it increases the maintenance work adding the necessity to check all related marks that they establish the proper order of tests and can cause that tests are carelessly littered accross all test files by contributors files without any logical order. Tracking issues in that case means having to following the trace through all `pytest_order` marks. That makes maintaining tests, quite difficult and tedious. Especially when it is unclear whether an error occurs cause test is outdated or component to be tested is broken and whether the test triggering the error is pointing to its source. In worst it is just triggering what should have been catched by an earlier test coded elsewhere in some other test file to be searched along `pytest_order`marks chain.

Reply:
I really do not agree with you.  But, the change is easy and arguing about this is a waste of time.